docs/tutorials/margins.ipynb

```diff
@@ -4,16 +4,377 @@
    "cell_type": "markdown",
```

Comment:
<p>explain more <code></code>right side of the catalog''</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astronomy-commons/lsdb/pull/244/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>